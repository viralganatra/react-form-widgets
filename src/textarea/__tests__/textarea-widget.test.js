import React from 'react';
import TextareaWidget from '../textarea-widget';

describe('TextareaWidget', () => {
    it('should render a single textarea widget', () => {
        const wrapper = shallow(<TextareaWidget />);

        expect(wrapper).toMatchSnapshot();
    });
});
