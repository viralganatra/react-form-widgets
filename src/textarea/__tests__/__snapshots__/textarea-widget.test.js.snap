exports[`TextareaWidget should render a single textarea widget 1`] = `<textarea />`;
